### ISSUE TITLE
[Improvement] Customizable GUI

### ISSUE THREAD

**Issue by anch0vie** [WasabiThumb/xclaim#22]
It would be nice if this plugin had a customizable GUI, such abilities to change icon, made a larger GUI menu, and delete something in the menu.

**Comment by WasabiThumb** [comment]
That would be a lot of work, how might this be helpful to servers?

**Comment by unofficialreece** [comment]
Sorry for the late reply. It is honestly just for cosmetics so that the menu can be highly customizable.

**Comment by WasabiThumb** [comment]
@rizeninkurtulusu Feel free to discuss how this feature might look here

**Comment by rizeninkurtulusu** [comment]
You've closed this. But, to me, I want to change gui's items & their custom model data to make resource pack supported cool designed menu. If you're going to add those options in your plugin, I promise to you, I'll make resource pack supported cool ui for your plugin free.

**Comment by WasabiThumb** [comment]
> You've closed this. But, to me, I want to change gui's items & their custom model data to make resource pack designed menu. If you're going to add those options in your plugin, I promise to you, I'll make resource pack designed for ui for your plugin free.

This issue is open. I closed #85 because it is a duplicate of this one, #22. Thank you for the feedback, I will consider it.

**Comment by WasabiThumb** [comment]
This is now being worked on at [this branch](https://github.com/WasabiThumb/xclaim/tree/feat/gui-refactor). See [here](https://github.com/WasabiThumb/xclaim/tree/feat/gui-refactor/src/main/resources/layouts) for what the layout configuration files look like at the moment, example below:
```xml
<layout>
    <row x="0" y="1" w="~" basis="CENTER">
        <slot id="0"/> <!-- NEW            -->
        <slot id="1"/> <!-- EDIT TRUST     -->
        <slot id="2"/> <!-- EDIT CHUNK     -->
        <slot id="3"/> <!-- RENAME CHUNK   -->
        <slot id="4"/> <!-- EDIT PERM      -->
    </row>
    <row x="0" y="2" w="~" basis="CENTER">
        <slot id="5"/> <!-- TRANSFER OWNER -->
        <slot id="6"/> <!-- CLEAR ALL      -->
        <slot id="7"/> <!-- DELETE         -->
        <slot id="8"/> <!-- VERSION        -->
        <slot id="9"/> <!-- EXIT           -->
    </row>
</layout>
```

ETA of a couple days.

**Comment by WasabiThumb** [comment]
Available in [1.15.0](https://github.com/WasabiThumb/xclaim/releases/tag/1.15.0). Sorry for the delay, merely a category 3 hurricane.
